packages/text-field/theme/lumo/vaadin-input-field-shared-styles.js

```diff
@@ -73,33 +64,6 @@ registerStyles(
       opacity: 0.5;
     }
 
-    [part='input-field'] {
-      border-radius: var(--lumo-border-radius-m);
-      background-color: var(--lumo-contrast-10pct);
-      padding: 0 calc(0.375em + var(--lumo-border-radius-m) / 4 - 1px);
-      font-weight: 500;
-      line-height: 1;
-      position: relative;
-      cursor: text;
-      box-sizing: border-box;
-    }
-
-    /* Used for hover and activation effects */
-    [part='input-field']::after {
-      content: '';
-      position: absolute;
-      top: 0;
-      right: 0;
-      bottom: 0;
-      left: 0;
-      border-radius: inherit;
-      pointer-events: none;
-      background-color: var(--lumo-contrast-50pct);
-      opacity: 0;
-      transition: transform 0.15s, opacity 0.2s;
-      transform-origin: 100% 0;
-    }
```

Comment:
This CSS is moved here:

https://github.com/vaadin/web-components/blob/91bfb8acc231ec7747bbebbfe2add5b83e647e00/packages/input-container/theme/lumo/vaadin-input-container-styles.js#L25-L38